src/library_sockfs.js

```diff
@@ -10,37 +10,35 @@ addToLibrary({
   },
   $SOCKFS__deps: ['$FS'],
   $SOCKFS: {
+#if expectToReceiveOnModule('websocket')
+    websocket_args: {},
```

Comment:
```suggestion
    websocket: {},
```
If we do need "args" in the name, should it be `websocketArgs`?

Reply:
There are some other existing props of this object that use snake_case (e.g. websocket_sock_ops) but I made this change anyway.

I think its good to have the name args in there.. since `websocket` on its own is a little misleading.. sounds like a noun.